lib/config/validation.ts

```diff
@@ -44,6 +44,7 @@ import {
 } from './validation-helpers/utils';
 
 const options = getOptions();
+const configFileNames = getConfigFileNames();
```

Comment:
they custom filenames are not configured at this stage 🤔

Reply:
Yes, instead of initializing a variable, `getConfigFileNames` fn should be called directly.

Then, for all calls we would get the custom filenames except when it's called from the config-validator module.